packages/List/src/List.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import ListItem, { Option, Member, OptionType } from './ListItem';
+
+import './list.scss';
+
+export interface ListProps extends React.ComponentProps<'ul'> {
+  /** Add a specific class to the Select. */
+  className?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** The option that is currently being focused or hovered */
+  focusedOption?: OptionType | null;
+  /** True for a compact appearance. */
+  isCompact?: boolean;
+  /** The List gains checkboxes beside each option
+   * to be able to select multiple options */
+  multiple?: boolean;
+  /** Specify the text to display when there are no results found */
+  noResultsText?: string;
```

Comment:
The list component doesn't know that it will be used by a parent who will have a search. In my opinion this property must be managed in the combobox